data/foods/beer_categories.json

```diff
@@ -1,16 +1,630 @@
 {
-  "description": "A list of beer categories.",
+  "description": "A list of beer categories and corresponding styles, according to the BJCP 2015 Style Guidelines",
```

Comment:
I would include a `"source"` property as well with a link to the PDF you referenced!

Reply:
added!